Security/Http/Authentication/AuthenticationFailureHandler.php

```diff
@@ -23,18 +24,31 @@ class AuthenticationFailureHandler implements AuthenticationFailureHandlerInterf
      */
     protected $dispatcher;
 
+    /**
+     * @var TranslatorInterface|null
+     */
+    protected $translator;
```

Comment:
please make it `private` (yes that's inconsistent but we don't want to provide such extension points anymore)